docs/standard/library-guidance/nuget.md

```diff
@@ -0,0 +1,99 @@
+| `PackageIconUrl`                   | `iconUrl`                  | A URL for an image to use as the icon for the package. URL should be HTTPS and the image should be 64x64 and have a transparent background.             |
+| `PackageProjectUrl`                | `projectUrl`               | A URL for the project homepage or source repository.             |
+| `PackageLicenseUrl`                | `licenseUrl`               | A URL to the project license. Can be the URL to the `LICENSE` file in source control.             |
+
+**✔️ CONSIDER** choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/en-us/nuget/reference/id-prefix-reservation).
+
+**✔️ CONSIDER** using the `LICENSE` file in source control as the `LicenseUrl`. For example, https://github.com/JamesNK/Newtonsoft.Json/blob/master/LICENSE.md
+
+> [!IMPORTANT]
+> A project without a license defaults to [exclusive copyright](https://choosealicense.com/no-permission/), making it impossible for other people to use.
+
+**✔️ DO** use an HTTPS href to your package icon.
+
+> Sites like NuGet.org run with HTTPS enabled and displaying a non-HTTPS image will create a mixed content warning.
+
+**✔️ DO** use a package icon image that is 64x64 and has a transparent background for best viewing results.
+
+## Pre-release packages
+
+NuGet packages with a version suffix are considered [pre-release](https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages). By default the NuGet Package Manager UI shows stable releases unless a user opts-in to pre-release packages, making pre-release packages ideal for limited user testing.
+
+```xml
+<PackageVersion>1.0.1-beta1</PackageVersion>
+```
+
+> [!NOTE]
+> A stable package cannot depend on a pre-release package. You must either make your own package pre-release, or depend on an older stable version.
+
+![NuGet pre-release package dependency](./media/nuget-prerelease-package.png "NuGet pre-release package dependency")
+
+**✔️ DO** publish a pre-release package when testing, previewing, or experimenting.
+
+**✔️ DO** publish a stable package when its ready so other stable packages can reference it.
+
+## Symbol packages
+
+NuGet supports [generating a separate symbol package](https://docs.microsoft.com/en-us/nuget/create-packages/symbol-packages) containing debug PDB files alongside the main package containing .NET assemblies. The idea of symbol packages is they're hosted on a symbol server and are only downloaded by a tool like Visual Studio on demand.
+
+Currently the main public host for symbols - [SymbolSource](http://www.symbolsource.org/) - doesn't support the portable PDBs created by SDK-style projects and symbol packages aren't useful.
+
+**✔️ CONSIDER** embedding PDBs in the main NuGet package.
```

Comment:
Is there any reason not to? Disk space is Cheapo

Reply:
Some libraries do still put PDBs in a symbol package and host on a dedicated symbol feed. .NET and ASP.NET packages for example do this.

If you don't have your own symbol server then embedding is best for most people.

Reply:
sadly, one library product i work on deliberately doesn’t provide pdbs because the business wants obfuscation of its IP